functions/sendMessagesWithMessageBird/main.js

```diff
@@ -0,0 +1,3 @@
+const sendMessagesWithMessageBird = require('./sendMessagesWithMessageBird');
+
+module.exports = sendMessagesWithMessageBird;
```

Comment:
put here the content of sendMessagesWithMessageBird thus having less unnecessary files